docs/frontmatter.md

```diff
@@ -54,6 +54,25 @@ project:
   open_access: true
 ```
 
+(composing-myst-yml)=
+:::{note} Composing multiple `.yml` files
+
+You may separate your frontmatter into multiple, composable files. To reference other files from your main `myst.yml` file, use the `extends` key with relative path(s) to the other configuration files:
+
+```yaml
+version: 1
+site: ...
+project: ...
+extends:
+  - ../macros.yml
```

Comment:
Does this work for externally-defined `.yml` files? (e.g. in a different github repository?)

If not, then would it be possible to download external files locally _as part of the build process_ in a way that would work as expected here?

Reply:
Currently the files must be available locally, from a relative path at build-time (they _can_ be outside the MyST project).

I agree we should be able to also put a url here, e.g.:
```yaml
extends:
  - https://example.com/my-other-myst.yml
```

We get into a discussion of this over here: https://github.com/executablebooks/mystmd/issues/336

But I think the url download/caching is a separate (smaller) PR.